doc/classes/int.xml

```diff
@@ -93,103 +91,102 @@
 			<return type="int" />
 			<param index="0" name="right" type="int" />
 			<description>
-				Returns the remainder after dividing two integers. This operation uses truncated division, which is often not desired as it does not work well with negative numbers. Consider using [method @GlobalScope.posmod] instead if you want to handle negative numbers.
+				Returns the remainder from dividing two integers. Uses truncated division, which returns negative numbers if the dividend is negative. Consider using [method @GlobalScope.posmod] if you want to get a positive remainder.
```

Comment:
I prefer the old text here. I prefer "after" instead of "from" since the primary result of a division is not the remainder. I prefer warning about the behavior with negative numbers, though the text could be reworded. Also, `posmod()` can give negative results if the divisor is negative, and we don't want people to think that this runs `abs()`, so that's white I wrote handle negative numbers instead of gives a positive remainder.

Reply:
Fair point about posmod. Just pushed a change, what do you think of my rewording?

Reply:
I think this is good 👍